app/controllers/api/measurement_sessions_controller.rb

```diff
@@ -34,7 +34,8 @@ def index
       data[:time_from] = Time.strptime(data[:time_from].to_s, '%s')
       data[:time_to] = Time.strptime(data[:time_to].to_s, '%s')
 
-      respond_with MobileSession.filtered_json(data, data[:limit], data[:offset])
+      respond_with sessions: MobileSession.filtered_json(data, data[:limit], data[:offset]),
+                   availableSessionsCount: MobileSession.filter(data).count
```

Comment:
In case you've used `amount` in the previous PR I'd suggest doing the same here

Reply:
This is a different value than in the previous PR. If it just says `amount` then it wouldn't be clear that it's not the number of fetched sessions, but rather the number of available sessions (cause we are fetching only 50).
And as a side note, I really don't like using the word `amount` for countable items 😉